clicmd: added help for the "virtualmedia umount" utility

Smith-x86 opened the pull request:
Added virtual media device name "USB2" for the "virtualmedia
umount" utility help.

Signed-off-by: Vladimir Kuznetsov <v.kuznetsov@yadro.com>

Smith-x86 committed:
clicmd: added help for the "virtualmedia umount" utility

Added virtual media device name "USB2" for the "virtualmedia
umount" utility help.

Signed-off-by: Vladimir Kuznetsov <v.kuznetsov@yadro.com>

AlexanderAmelkin reviewed (changes requested):
Please update according to the latest task update.

Smith-x86 committed:
clicmd: Rename Virtual Media devices from USB to Device

Renamed Virtual Media device names from USB to Device:
- Device0 instead of USB0;
- Device1 instead of USB1;
- Device2 instead of USB2.

Signed-off-by: Vladimir Kuznetsov <v.kuznetsov@yadro.com>

Smith-x86 commented:
> Please update according to the latest task update.

Done
